src/app/components/videochat/jh-video-chat.directive.js

```diff
@@ -122,7 +122,11 @@
           }
           // If current one is still speaking, there is no need to change
           if ( !(current && current.getSpeaking()) ) {
-            vm.highlight.current = FeedsService.speakingFeed() || current || FeedsService.findMain();
+            if (FeedsService.speakingFeed() && FeedsService.speakingFeed().getVideoSubscription()){
```

Comment:
Although it's almost impossible, there is a chance than the two consecutive calls to `FeedService.speakingFeed()` return different feeds. Better use a variable, like
```
var speaking = FeedService.speakingFeed();
if (speaking && speaking.isEnabled("video")) {
```